src/domain/accounts/index.types.d.ts

```diff
@@ -11,6 +11,27 @@ type DepositFeeRatio = number & { readonly brand: unique symbol }
 
 type ContactAlias = string & { readonly brand: unique symbol }
 
+type AccountLimitsArgs = {
+  level: AccountLevel
+  accountLimits?: {
+    intraLedger: {
+      level: {
+        [l: number]: UsdCents
```

Comment:
What about level 2?

Reply:
the syntax says there can be any limits (as is), as long as the key is a number, this will pass.